src/components/TransactionsTab/TransactionsTab.tsx

```diff
@@ -2,7 +2,7 @@ import { useCallback, useEffect, useRef, useMemo, useState } from "react";
 import { useTranslation } from "react-i18next";
 
 import { TokenInfo } from "@airswap/types";
-import { getEtherscanWalletURL } from "@airswap/utils";
+import { getEtherscanURL } from "@airswap/utils";
```

Comment:
`getEtherscanWalletURL` is for getting wallet url.
`getEtherscanURL` is for getting a transaction url.

Reply:
but you're right, I can't just swap `getEtherscanWalletURL ` for `getEtherscanURL`...